library/books-api-v2/Containerfile

```diff
@@ -0,0 +1,10 @@
+FROM registry.access.redhat.com/ubi8/nodejs-14:1
+
+COPY . /opt/app-root/src/
+
+RUN cd /opt/app-root/src && \
+    npm install
```

Comment:
I would recommend:

- add package-lock.json as well
- execute `npm ci --production` instead of `npm install` (which will update dependencies)